installation/proc-install-rhdh-helm.adoc

```diff
@@ -27,17 +27,34 @@ For more information about creating a project in OpenShift, see link:https://doc
 . From the *Developer Catalog* panel, click *Helm Chart*.
 . Search for _Developer Hub_ in the search bar and select the *{product}* card.
 . Click *Create*.
-. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.*, and adjust the other values if needed.
+. Use one of the following views to edit the Helm configuration:
 
-. Alternatively, copy the OpenShift router host (for example: `apps.<clusterName>.com`) to `global.clusterRouterBase`, and adjust other values if needed, such as `global.clusterRouterBase: apps.example.com`. 
+** Using *Form view* 
++
+.. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) and paste it in the *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.* field, and edit the other values if needed.
+.. Click *Create* and wait for the database and {product} to start.
+
+** Using *YAML view* 
++
+.. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) and paste it in the `global.clusterRouterBase` parameter value as shown in the following example:
++
+[source,yaml]
+----
+global:
+  auth:
+    backend:
+      enabled: true
+  clusterRouterBase: apps.ci-In-6xvy78k-76ef8.origin-ci-int-aws.dev.rhcloud.com
```

Comment:
Minor suggestion - Instead of use a _real_ value for the `clusterRouterBase`, you can use the same value described in the documentation `apps.<clusterName>.com`. It makes consistent and clear the documentation.

Reply:
done